src/components/Recommended/Recommended.js

```diff
@@ -2,10 +2,10 @@ import React from 'react'
 import RecommendedAlbum from '../RecommendedAlbum/RecommendedAlbum'
 import './Recommended.css'
 
-const Recommended = ({ recommendedData }) => {
+const Recommended = ({ trendingData }) => {
     
-    if (recommendedData) {
-        const recommendedAlbums = recommendedData.results.map(item => <RecommendedAlbum key={item.id} id={item.id} title={item.title} coverImg={item.cover_image}/>)
+    if (trendingData) {
+        const recommendedAlbums = trendingData.results.map(item => <RecommendedAlbum key={item.id} id={item.id} title={item.title} coverImg={item.cover_image}/>)
```

Comment:
I gotta ask why you changed this to 'trending' - I think when I designed it I was thinking this component could be reused for "Trending"(`/trending`) AND "Recommended for you" (`/discover`) albums.  I think `recommendedData` would best to accommodate both types?

Reply:
I changed it because I thought we could improve the app by only fetching the trending data on visiting the page once, and just keeping it in state since this data would never change in the short term. If we change it to how you described it we would need to fetch it every time we visit `/trending` and change the state. The `discover` feature would be random, so this would need to fetch all the time. We can talk about how to proceed, I don't mind changing it back if it makes sense.